README.md

```diff
@@ -70,8 +71,11 @@ import {
 Start the stub server. Argument `args` values are passed directly to specmatic jar executable.<br />
 *Note: This replaces `startStub` method which is deprecated*
 
+`setExpectationJson(stubResponse: any, stubServerBaseUrl?: string): Promise<boolean>` <br />
+Set stub expectation from a static JSON object. Stub should be running before invoking this method.<br />
+
 `setHttpStubExpectations(stubPath: string, stubServerBaseUrl?: string): Promise<boolean>` <br />
```

Comment:
I'm wondering if we could simply overload this method with a the first argument, instead of a string but some fixed type for mapping the http-request and response.